src/JMS/Serializer/GenericDeserializationVisitor.php

```diff
@@ -162,7 +162,7 @@ public function visitProperty(PropertyMetadata $metadata, $data, Context $contex
     {
         $name = $this->namingStrategy->translateName($metadata);
 
-        if (null === $data || ! array_key_exists($name, $data)) {
+        if (null === $data || ! is_array($data) || ! array_key_exists($name, $data)) {
```

Comment:
@schmittjoh after nearly a year, I fix it ...

thank you @Gladhon, for notifying me